Update README for release 1.4 (#175)
memgraph/mage: README.md

| ![graph_input](img/graph_input.png) | ![graph_output](img/graph_output.png) |

## MAGE Spells

> Graph neural network-based node classification module

Reviewer: Error?

Author: yes

| [json_util](https://memgraph.com/docs/mage/query-modules/python/json-util)                                       | Python | A module for loading JSON from a local file or remote address.                                                                                                                                                                    |
| [katz_centrality](https://memgraph.com/docs/mage/query-modules/cpp/katz-centrality)                              | C++    | Katz centrality is a centrality measurement that outputs a node's influence based on the number of shortest paths and their weighted length.                                                                                      |
| [katz_centrality_online](https://memgraph.com/docs/mage/query-modules/cpp/katz-centrality-online)                | C++    | Online implementation of the Katz centrality. Outputs the approximate result for Katz centrality while maintaining the order of rankings.                                                                                         |

> | [kmeans](https://memgraph.com/docs/mage/query-modules/python/kmeans)                                             | Python | Algorithm to cluster given data.                                                                                                                                                                                                  |

Reviewer: Grammar
```suggestion
| [kmeans](https://memgraph.com/docs/mage/query-modules/python/kmeans)                                             | Python | An algorithm for clustering given data.                                                                                                                                                                                                  |
```

> | [link_prediction_gnn](https://memgraph.com/docs/mage/query-modules/python/link-prediction-with-gnn)              | Python | Module for predicting links in the graph by using graph neural networks.                                                                                                                                                          |

Reviewer: ```suggestion
| [link_prediction_gnn](https://memgraph.com/docs/mage/query-modules/python/link-prediction-with-gnn)              | Python | A module for predicting links in graphs using graph neural networks.                                                                                                                                                          |
```

| [max_flow](https://memgraph.com/docs/mage/query-modules/python/max-flow)                                         | Python | An algorithm for calculating maximum flow through a graph using capacity scaling                                                                                                                                                  |

> | [node-classification_with_gnn](https://memgraph.com/docs/mage/query-modules/python/node-classification-with-gnn) | Python | Graph neural network-based node classification module.                                                                                                                                                                            |

Reviewer: ```suggestion
| [node-classification_with_gnn](https://memgraph.com/docs/mage/query-modules/python/node-classification-with-gnn) | Python | A graph neural network-based node classification module.                                                                                                                                                                            |
```
